README.md

```diff
@@ -1,2 +1,87 @@
 # VR-Volumeviewer
-A simple viewer for visualizing volumetric datasets
+A simple viewer for visualizing volumetric datasets. It supports color mapping using transfer functions to map specific values of data to a scalar color. The view camera is fully interactive, and it works on multiple systems (Windows and Linux).
+This tool is fully integrated with [MinVR](https://github.com/MinVR/MinVR), a software that facilitates the data visualization on multiple displays devices (desktop, VR HMDs,cave-systems).
+
+### Requirements
+
+* [Cmake 3.9 or greater ](https://cmake.org/)
+* [VR-Imgui](https://github.com/brown-ccv/VR-imgui)
+* [OpenCV](https://github.com/opencv/opencv)
```

Comment:
is opencv still needed since the repo uses the opencv submodule?

Reply:
In fact, those submodules need to be removed. the super build now it's 100% managed in cmake's auto download